### ISSUE TITLE
launchdarkly_feature_flag_environment errors when fallthrough bucket_by is null

### ISSUE THREAD

**Issue by rayward** [launchdarkly/terraform-provider-launchdarkly#120]
When the `bucket_by` arg of the `fallthrough` block of `launchdarkly_feature_flag_environment` is explicitly set to `null`, then this error is seen:

```
Error: failed to update flag "MyFeature" in project "default": 400 Bad Request: {"code":"invalid_request","message":"Percentage rollout must be bucketed by a non-empty attribute name"}
```

As this arg is optional, it should be valid to specify `null`, the provider should then omit this param when patching the resource to configure the rollout to use the default bucketing method (by user key).

**Comment by ldhenry** [comment]
Hey @rayward,

Thanks for bringing this to our attention. The team has started working on a fix and we should have something out soon.

Cheers,
Henry

**Comment by lucywyman** [comment]
This should be fixed in 2.9.3 of the provider! Let us know if you're still having this issue.

**Comment by rayward** [comment]
@lucywyman everything is working great now with the release, thanks!

Tested these scenarios and all behaved as expected:
- omitting `bucket_by` arg 
- setting `bucket_by` to `null`
- setting `bucket_by` to an attribute name
